src/NHSD.BuyingCatalogue.Identity.Api/Views/Consent/Index.cshtml

```diff
@@ -1,8 +1,8 @@
 ﻿@model NHSD.BuyingCatalogue.Identity.Api.ViewModels.Consent.ConsentViewModel
 
 @{
-    ViewData["PageTitle"] = "User agreement";
-    ViewData["PageDescription"] = "PLEASE READ THESE TERMS CAREFULLY BEFORE USING THE CATALOGUE";
+    ViewData["PageTitle"] = "Buying Catalogue End User Agreement";
+    ViewData["PageDescription"] = "Please read this agreement carefully before using the NHS Digital Buying Catalogue";
 }
 
 <style>
```

Comment:
I am not sure about these styles being in her

Reply:
For the styles do you think its better pulling them out into a seperate css class?

Reply:
@kat1906  ^

Reply:
I personally don't mind as it is unlikely they will get used elsewhere and the li style should just be applied on this page

Reply:
Fair enough. Keep it in then 👍